src/plugins/condition/components/ConditionSet.vue

```diff
@@ -34,8 +34,10 @@
         </div>
     </section>
     <TestData :is-editing="isEditing" />
-    <ConditionCollection :is-editing="isEditing"
-                         @conditionSetResultUpdated="updateCurrentOutput" />
+    <ConditionCollection
```

Comment:
I like the change but had thought our standard was to put the first attribute on the first line and indent all subsequent attributes the same as the first. To fix the linting issue you would just need to move the closing bracket to its own line. Again, I like your pattern better and would agree this is more readable.

Reply:
Yeah the only linting error is the closing bracket should be in a newline. I just prefer in this case to put all attributes on their own line. Its more natural for me to do so, and especially in this case the tag name is so long it just bothers me to indent the next line so far over.

We decided to not to enforce a strict standard, but allow the user to do what feels more comfortable to them. You can leave the first attribute on the first line if you want, but its not required. If you do that the next attribute lines up with the first attribute for indention. I believe what you cannot do is have multiple attributes in the same line.

Reply:
I guess I assumed that doing it the way you did would also be a lint violation. Knowing it's not, I'll adopt your pattern going forward.